Assets/Scenes/GameBoardMain/Scripts/InstantiateGemPrefabs.cs

```diff
@@ -6,9 +6,16 @@ public class InstantiateGemPrefabs : MonoBehaviour
     public Transform Prefab;
     void Start()
     {
+
+    	// Get board from GameState
+    	var gameState = GameObject.Find("GameState").GetComponent<GameState>();
+    	var board = gameState.Board;
+    	Debug.LogFormat("Board first row: [{0}, {1}, {2}, {3}, {4}]",
+    		board.GemAt(0, 0), board.GemAt(1, 0), board.GemAt(2, 0), board.GemAt(3, 0), board.GemAt(4, 0));
+
```

Comment:
I believe this is how you would get access to the GameState Board abstraction from any other Game Object.
I couldn't see this printed, but I think this is because this Script is not currently attached to the board game object in the Scene.

The idea here is that you should be able to get what gems should be displayed in the Board Game Objects from the state.

Reply:
Okay. Let's see if this ends up working. I wonder what happens when there are multiple board game objects at that location? For example, the tile and the gem. Will it display a collection of them that we can select by name? Let's get it attached and see.

Reply:
The idea is that we have a `gameState.Board` object that represents the board in memory in the simplest possible way, that is, a color on each coordinate from [0, 0] to [5, 5].

The state is duplicated in the scene game objects. The scene may have a tile and the gem, and many other things for details, shadows, particles, etc. The challenge is in keeping the state in synch:

 * One function that can re-build the scene from a given `gameState.Board` object.
 * When the scene is modified (maybe by the effects of dragging gems around, or a card play), then the `gameState.Board` will have to be updated.

And then, every time we update the `gameState` (Board, Player1, Player2 or InvocationArea) it will be easy to send that state through the network to rebuild it on the client side.

Hope this makes more sense!